setup.py

```diff
@@ -74,6 +74,7 @@ def read_requirements(path):
     license="MIT",
     python_requires=">=3.9",
     install_requires=requirements,
+    extras_require={"benchmark": ["lm_eval[math,ifeval,sentencepiece]==0.4.4"]},
```

Comment:
Just here to avoid regular miners/validators from needing to install?

Reply:
Yup!